io/shared/src/main/scala/fs2/io/process/ProcessBuilder.scala

```diff
@@ -67,17 +70,29 @@ sealed abstract class ProcessBuilder private {
   /** @see [[workingDirectory]] */
   def withCurrentWorkingDirectory: ProcessBuilder
 
+  /** @see [[outputMode]] */
+  def withOutputMode(outputMode: ProcessOutputMode): ProcessBuilder
+
   /** Starts the process and returns a handle for interacting with it.
     * Closing the resource will kill the process if it has not already terminated.
     */
   final def spawn[F[_]: Processes]: Resource[F, Process[F]] =
     Processes[F].spawn(this)
 }
 
+sealed trait ProcessOutputMode
+object ProcessOutputMode {
```

Comment:
These look like the right set of options! We need to support `stdin`, `stdout`, and `stderr` to be configured separately.